files/install-deps-worker.yaml

```diff
@@ -22,11 +31,15 @@
           - python3-bugzilla # python-bugzilla (not bugzilla) on PyPI
           - python3-backoff # Bugzilla class
           #- python3-flask-restx # needs Fedora 32
+          - dnf-utils
+          - python3-pip
+          - make
         state: present
     - name: Install pip deps
       pip:
         name:
           - git+https://github.com/packit-service/sandcastle.git
           - sentry-sdk==0.14.4
           - flask-restx
+          - git+https://github.com/packit-service/packit.git
```

Comment:
As [Hunor pointed out in the research](https://github.com/packit-service/research/pull/52#discussion_r448181457) we want to install packit from `master` (branch) when building for stage and from `stable` when building for prod.

This playbook (`install-deps-worker.yaml`) is run by both, `Dockerfile.worker` and `Dockerfile.worker.prod` so we need to somehow pass the information about whether we want packit from `master` or `stable` into this playbook and act according to it.

The same applies to installing packit dependencies.

Reply:
implemented